app/code/Bulbulatory/Recommendations/view/frontend/web/js/view/checkout/cart/totals/discount.js

```diff
@@ -0,0 +1,16 @@
+define(
+    [
+       'Bulbulatory_Recommendations/js/view/checkout/summary/discount-fee'
+],
+function (Component) {
+    'use strict';
+    return Component.extend({
+        /**
+         * @override
+         */
+        isDisplayed: function () {
+            return true;
+        }

```

Comment:
Returning here `true` means that the discount will always be visible, even if it's not applied due to lack of recommendations. So it will display something like `Recommendation Discount    0,00 USD`. It doesn't affect the cart value nor total price so there's no point in displaying it. It's better to return true/false based on the discount value - if there is a discount, return true, if not then false.

Reply:
@TM18

> Returning here true means that the discount will always be visible, even if it's not applied due to lack of recommendations. So it will display something like Recommendation Discount 0,00 USD. It doesn't affect the cart value nor total price so there's no point in displaying it. It's better to return true/false based on the discount value - if there is a discount, return true, if not then false.

Fixed!